docs/source/implementations/sb_bagging.rst

```diff
@@ -0,0 +1,151 @@
+.. _implementations-sb_bagging:
+
+======================================================
+Sequentially Bootstrapped Bagging Classifier/Regressor
+======================================================
+
+In sampling section we have shown that sampling should be done by Sequential Bootstrapping.
+SequentiallyBootstrappedBaggingClassifier and SequentiallyBootstrappedBaggingRegressor extend sklearn's BaggingClassifier/Regressor by using Sequential Bootstrapping instead of random sampling.
+
+In order to build indicator matrix, Triple Barrier Events and price bars used to label training data set should be passed as parameters.
+
+
+Sequentially Bootstrapped Bagging Classifier
+============================================
+
+.. py:class:: SequentiallyBootstrappedBaggingClassifier(SequentiallyBootstrappedBaseBagging, BaggingClassifier,
```

Comment:
Should these examples be replaced by the auto gen functions? @imcu gave a good example in the CV documentation PR which is now in dev.